ajax/libs/iphone-inline-video/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "iphone-inline-video",
+  "filename": "iphone-inline-video.min.js",
+  "version": "2.2.2",
+  "description": "Make videos playable inline on the iPhone (prevents automatic fullscreen)",
+  "homepage": "https://npm.im/iphone-inline-video",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bfred-it/iphone-inline-video.git"
+  },
+  "author": "Federico Brigante <github@bfred.it> (bfred.it)",
+  "keywords": [
+    "browser",
+    "iphone",
+    "ios",
+    "play",
+    "video",
+    "inline",
+    "fullscreen",
+    "full screen",
+    "ipod",
+    "autoplay",
+    "multiple",
+    "audio",
+    "sound",
+    "canvas",
+    "gif"
+  ],
+  "npmName": "iphone-inline-video",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.!(common-js|es-modules)"
```

Comment:
The current pattern still can get the `iphone-inline-video.common-js.js` and `iphone-inline-video.es-modules.js` files.
Let's use `**/!(*common-js|*es-modules|*node)*` exclude the `iphone-inline-video.es-modules.js` and `iphone-inline-video.common-js.js` and `iphone-inline-video.node.js`in v1.0.1 & v1.0.2.
Thanks